source/features/netiquette.tsx

```diff
@@ -26,40 +26,49 @@ function getCloseDate(): Date {
 
 const threeMonths = toMilliseconds({days: 90});
 
-function addConversationBanner(newCommentField: HTMLElement): void {
+export function shouldDisplayNotice(): boolean {
 	if (!isClosedOrMerged()) {
-		return;
+		return false;
 	}
 
 	const closingDate = getCloseDate();
-	if (timeAgo(closingDate) < threeMonths) {
-		return;
-	}
+	return timeAgo(closingDate) > threeMonths;
+}
 
+export function getNoticeText(): JSX.Element {
+	const closingDate = getCloseDate();
 	const ago = <strong>{twas(closingDate.getTime())}</strong>;
 	const newIssue = <a href={buildRepoURL('issues/new/choose')}>new issue</a>;
+	return (
+		<>
+			This {pageDetect.isPR() ? 'PR' : 'issue'} was closed {ago}. Please consider opening a {newIssue} instead of leaving a comment here.
+		</>
+	);
+}
 
+function addConversationBanner(newCommentField: HTMLElement): void {
 	newCommentField.before(
 		createBanner({
+			icon: <InfoIcon/>,
 			classes: 'p-2 m-2 text-small color-fg-muted border-0'.split(' '),
-			text: (
-				<div className="d-flex flex-items-center gap-2">
-					<InfoIcon className="m-0"/>
-					{/* TODO: Drop any after https://github.com/frenic/csstype/issues/177 */}
-					<span style={{textWrap: 'balance'} as any}>
-						This {pageDetect.isPR() ? 'PR' : 'issue'} was closed {ago}. Please consider opening a {newIssue} instead of leaving a comment here.
-					</span>
-				</div>
-			),
+			text: getNoticeText(),
```

Comment:
To avoid duplicating a lot of logic and to avoid awkward if/else in this feature, I tried making multiple components the best I could. Not super great 😂 but reasonable dry

Reply:
Mobile is a bit broken at the moment:

<img width="493" alt="Screenshot 11" src="https://user-images.githubusercontent.com/1402241/227766822-f36abb90-39f2-4302-8edd-70c4a61bdd96.png">

Reply:
Spacing fixed

## Desktop


<img width="434" alt="Screenshot 2" src="https://user-images.githubusercontent.com/1402241/228140912-5d58e004-ecbb-4fe4-8adb-c4204b4c3746.png">


## Mobile

<img width="482" alt="Screenshot 1" src="https://user-images.githubusercontent.com/1402241/228140936-23f28b47-4983-48fc-8cae-8286d4881edd.png">